recipes/json-glib/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
```

Comment:
Can you rename this to `test_package.c`?
This way, we test whether the c++ runtime is required (eventually by a dependency) or not.